src/appwindow.cpp

```diff
@@ -415,27 +430,30 @@ void AppWindow::openTab(const QString &path)
         }
     }
 
-    auto newWindow = new MainWindow(path, getNewUntitledIndex(), preferencesWindow, this);
```

Comment:
Instead of passing only `PreferenceWindow`, we pass Complete `AppWindow` and it has a getter `getPreferenceWindow()` which can be used to initialize the messagelogger.

Reply:
Doing changes here only creates more conflicts, mixing up pull requests, and making conversations hard to find.